resources/fetchers/ecr_fetcher.go

```diff
@@ -86,13 +93,26 @@ func (f *ECRFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata
 			}
 		}
 	}
-
 	return nil
 }
 
 func (f *ECRFetcher) describePodImagesRepositories(ctx context.Context, podsList *v1.PodList, describer PodDescriber) ([]ecr.Repository, error) {
+	regionToReposMap := getAwsRepositories(podsList, describer)
+	f.log.Debugf("sending pods to ecrProviders: %v", regionToReposMap)
+	awsRepositories := make([]ecr.Repository, 0)
+	for region, repositories := range regionToReposMap {
+		// Add configuration
+		describedRepo, err := describer.Provider.DescribeRepositories(ctx, f.awsConfig, repositories, region)
+		if err != nil {
+			f.log.Errorf("could not retrieve pod's aws repositories for region %s: %w", region, err)
+		}
+		awsRepositories = append(awsRepositories, describedRepo.Repositories...)
```

Comment:
if there's an error it will still add the decribedRepo into awsRepositories. Is this by design?